.github/workflows/release.yml

```diff
@@ -31,63 +23,104 @@ jobs:
     strategy:
       matrix:
         include:
-        - image: ubuntu-20.04
-          platform: linux-amd64
-        - image: ubuntu-20.04
-          platform: linux-arm64
-        - image: macos-12
-          platform: darwin-arm64
-        - image: macos-12
-          platform: darwin-amd64
-        - image: windows-2022
-          platform: windows-amd64
-
+          - image: ubuntu-20.04
+            platform: linux-amd64
+          - image: ubuntu-20.04
+            platform: linux-arm64
+          - image: macos-12
+            platform: darwin-arm64
+          - image: macos-12
+            platform: darwin-amd64
+          - image: windows-2022
+            platform: windows-amd64
     runs-on: ${{ matrix.image }}
     steps:
       - uses: actions/setup-go@v3
         name: Install Go
         with:
           go-version: ${{ env.GO_VER }}
+
       - uses: actions/checkout@v3
         name: Checkout Fabric CA Code
-      - name: Install linux aarch64 compiler
-        run: sudo apt-get -y install gcc-aarch64-linux-gnu
+
+      - name: Install GCC cross-compilers
         if:  ${{ contains(matrix.platform, 'linux') }}
+        run: |
+          sudo apt-get -y install gcc-aarch64-linux-gnu
+          sudo apt-get -y install gcc-x86-64-linux-gnu
+
       - run: make dist/${{ matrix.platform }}
         name: Compile Binary and Create Tarball
+        env:
+          BASE_VERSION: ${{ github.ref_name }}
+
       - uses: actions/upload-artifact@v3
         name: Publish Release Artifact
         with:
-          name: hyperledger-fabric-ca-${{ matrix.platform }}-${{ inputs.release }}.tar.gz
-          path: release/${{ matrix.platform }}/hyperledger-fabric-ca-${{ matrix.platform }}-${{ inputs.release }}.tar.gz
+          name: hyperledger-fabric-ca-${{ matrix.platform }}-${{ github.ref_name }}.tar.gz
+          path: release/${{ matrix.platform }}/hyperledger-fabric-ca-${{ matrix.platform }}-${{ github.ref_name }}.tar.gz
+
+  build-and-push-image:
+    runs-on: ubuntu-latest
```

Comment:
Don't think it matters here, but we usually try to stick to a certain version (currently ubuntu-20.04):
- more consistency
- less confusion
- less chance of behavior changes or breaks when 'latest' changes without our knowledge
- for binaries, less chance of accidentally building binaries on a later version that won't work with earlier ubuntu versions